### ISSUE TITLE
Relationship between data and learning speed

### ISSUE THREAD

**Issue by imsoncod** [tanreinama/text2text-japanese#2]
Why is the learning rate the same as the amount of data increases?

**Comment by tanreinama** [comment]
Is it the case when warmup_steps>0?
text2text is a program for fine tuning and does not assume only a specific corpus.
Therefore, the user needs to optimize the learning parameters according to the size and characteristics of the corpus.
The default value for warmup_steps is 0, because learning is almost stable with the pre-trained GPT2 model, but I've left an option in case there is a corpus that is too different from the language model.
In most cases, warmup_steps can be 0, but if you want to use a non-zero value, set an appropriate value along with the learning rate according to the size and characteristics of the corpus.

**Comment by imsoncod** [comment]
Is it necessary to change warmup_steps to train QtoA-type question-and-answer data?

**Comment by tanreinama** [comment]
In most cases it is not needed.

**Comment by imsoncod** [comment]
Thank you 😀
